### DISCUSSION TITLE
Anyone want to start up a Discord server?

### DISCUSSION THREAD

**Discussion by GodotCommunityGames** [Godot-Community-Games-Org/.github#3] [General]
These discussion sections are good for message board style communication, but having a Discord server would probably be good for realtime chat style communication. And numerous people have already asked if we're starting one up.

I've never really used Discord and don't know anything about it. Ideally one of you that is all into Discord and knows what features make a good Discord server would want to start up a "Godot Community Games" discord.

And I mean also I don't want to be like overlording over this whole thing and running everything, I just want to be a community member with the rest of you so that this whole project is run by the group and not by me just because I happened to have the idea and start it up, so having someone else in the community run the discord would be good anyway.

So anyone interested in running a Discord server for this and getting the community onboarded to it, post in here. Thanks!

**Comment by Yonodactyl** [comment]
The unfortunate truth that I see with a lot of projects is that someone does need to spearhead a little bit if the process - if not forever, at least at first. I think you're doing great, and I agree it's more fun to be an individual contributor vs a manager.

I think if you set that standard from the beginning, you should be able to enjoy instead of delegating. You'll probably need to establish some mods in the discord to make sure things maintain a decorum; it can get wild pretty quick.

### More thoughts: 
At the end of the day, we all just want to make games. I think as long as we are respectful and communicate things, we should be good. Also, setting some code ownership might be good - what I mean is that projects should require at least one approval from another dev to merge. If the project ends up being only a solo dev thing (no one joined in the allotted brainstorming phase), then that dev works in full autonomy, but if there is more than one there needs to be a defining process to the flow otherwise it can get messy really quick!

**Reply by GodotCommunityGames** [reply]
Yeah I mean I'm fine running this Github org and taking a leading role setting the whole community up before settling back mostly into a simply community member role. But I legit know nothing about Discord or setting one up or what makes a good discord or what features they even have or what running one entails, and so I don't have an interest in running one, but I'd be happy to join a community discord that some more Discord-knowledgable member sets up. There's lots of people who love Discord, meanwhile I know nothing about Discord, so I think it makes a lot more sense for someone who actually has experience using Discord to set it up.

**Comment by StrawberrySmoothieDev** [comment]
Already got one w most of us here: https://discord.gg/uQ3fe43hdc

**Reply by AnonymousAggregator** [reply]
joined
